pyinfra/facts/files.py

```diff
@@ -276,6 +276,12 @@ class Sha256File(HashFileFactBase, digits=64, cmds=["sha256sum", "shasum -a 256"
     """
 
 
+class Sha384File(HashFileFactBase, digits=96, cmds=["sha384sum", "shasum -a 384"]):
```

Comment:
I just remembered why I left this PR as a draft :slightly_smiling_face: 

I was unsure whether to add `sha384` to `cmds` - the other facts have the respective executables. But the only executables called `shaXXX` that I could find are symlinks registered by `hashalot`, and they seem to do a different thing...

@Fizzadar Any idea?

Reply:
The non-sum ending variants are a macOS (possibly BSD) thing! By default macOS has `sha384` but not `sha384sum`, both are identical in behavior. So yes let's add them in 👍

Reply:
Thank you! I've added `sha384`.